packages/pn-personafisica-webapp/src/navigation/RouteGuard.tsx

```diff
@@ -12,6 +13,16 @@ const RouteGuard = () => {
   const [params] = useSearchParams();
   const { sessionToken } = useAppSelector((state: RootState) => state.userState.user);
 
+  useEffect(() => {
```

Comment:
Pf doesn't need to store the aar, because the login process return the aar in the url. Restore the old version